test/lib/index.test.ts

```diff
@@ -19,3 +19,20 @@ test("Valid TreeParsing", async () => {
 
   expect(resDepGrah.equals(fixtureDepGraph)).toBeTruthy();
 });
+
+test("Valid lein deps tree parsing", async () => {
+  const validTreeContent = fs.readFileSync(
+    path.resolve(__dirname, "../fixtures/lein-tree"),
+    "utf-8"
+  );
+  const fixtureDepGraphContent = JSON.parse(
+    fs.readFileSync(
+      path.resolve(__dirname, "../fixtures/valid-lein-tree-dep-graph.json"),
+      "utf-8"
+    )
+  ) as DepGraphData;
+  debugger;
```

Comment:
What is `debugger` here? I have never seen it :-)

Reply:
neither have I